spec/models/project_spec.rb

```diff
@@ -56,4 +57,44 @@
 
     expect(project).to be_valid
   end
+
+  context "project and users exist with tagged skills and interests" do
+    let!(:project) do
+      FactoryGirl.create(:project, skill_list: "ruby, rails,
+      css, javascript, technical writing, video production",
+      interest_list: "film, fitness, technology")
+    end
+    let!(:irrelevant_user) { FactoryGirl.create(:user) }
+    let!(:relevant_user) do
+      FactoryGirl.create(:user, skill_list: "ruby, rails")
+    end
+    let!(:most_relevant_user) do
+      FactoryGirl.create(:user, skill_list: "ruby, python",
+      interest_list: "film, big data")
+    end
+    describe "#recommended_users" do
+      it "returns and orders the users that best match the skills and interests
+        of the project" do
+        expect(project.recommended_users[0][0]).to eq most_relevant_user
+        expect(project.recommended_users[1][0]).to eq relevant_user
+      end
+
+      it "does not return users that have no matches" do
+        expect(project.recommended_users.include?([irrelevant_user]) == false)
+      end
+    end
+
+    describe "#users_matches" do
+      it "returns a user's skills and interests that match the project's" do
+        expect(project.users_matches(most_relevant_user).include?(
+          {"interest" => "film"}))
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than `project.users_matches(most_relevant_user).include?(`.<br>Space inside { missing.<br>Space inside } missing.